src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,6 +72,9 @@ private void configureButtonBindings() {
     OI.Driver.getAlignBackPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(180), swerveSubsystem));
     OI.Driver.getAlignLeftPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(90), swerveSubsystem));
     OI.Driver.getAlignRightPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(-90), swerveSubsystem));
+    OI.Operator.getConfirmBuddyBalanceButton().whileTrue(new InstantCommand(buddyBalanceSubsystem::deployBuddyBalance, buddyBalanceSubsystem));
+    OI.Operator.getUpBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::retrieveBuddy, buddyBalanceSubsystem));
+    OI.Operator.getDownBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::releaseBuddy, buddyBalanceSubsystem));
```

Comment:
I thought that the intention was that the B button would not do anything other than enable the buddy balance (No moving parts in case it is pressed accidentally), and the POV buttons would "cycle" the balance up/down through the 3 positions. It seems that's not exactly what's happening? Or am I reading something wrong

Reply:
There will only be 2 positions because the way the buddy balance is being designed, once it's deployed it can't retract itself. The issue with the B button should be fixed in the next commit though